src/Components/Positions/Form/FormPositions.js

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useState } from 'react';
+import styles from './form.module.css';
+
+const FormPositions = () => {
+  const [jobTitle, setJobTitle] = useState ("");
+  const [clientId, setClientId] = useState ("");
+  const [companyName, setCompanyName] = useState ("");
+  const [jobDescription, setjobDescription] = useState ("");
+  const [city, setCity] = useState ("");
+  const [country, setCountry] = useState ("");
+  const [datePosted, setDatePosted] = useState ("");
+  const [closingDate, setClosingDate] = useState("");
+
+  const onChangeJobTitle = (event) => {
+    setJobTitle(event.target.value);
+  };
+  const onChangeClientId = (event) => {
+    setClientId(event.target.value);
+  };
+  const onChangeCompanyName = (event) => {
+    setCompanyName(event.target.value);
+  };
+  const onChangeJobDescription = (event) => {
+    setjobDescription(event.target.value);
+  };
+  const onChangeCity = (event) => {
+    setCity(event.target.value);
+  };
+  const onChangeCountry = (event) => {
+    setCountry(event.target.value);
+  };
+  const onChangeDatePosted = (event) => {
+    setDatePosted(event.target.value);
+  };
+  const onChangeClosingDate = (event) => {
+    setClosingDate(event.target.value);
+  };
+
+  const params = new URLSearchParams(window.location.search);
+  const PosId = params.get('id');
+
+  useEffect(()=> {
+    if(PosId){
+      fetch(`${process.env.REACT_APP_API}/api/positions/id/${PosId}`)
+        .then((response) => {
+          if (response.status !== 200) {
+            return response.json().then(({ ErrMessage }) => {
+              throw new Error(ErrMessage);
+            });
+          }
+          return response.json();
+        })
+        .then((response) => {
+          setJobTitle(response.jobTitle);
+          setClientId(response.clientId);
+          setCompanyName(response.companyName);
+          setjobDescription(response.jobDescription);
+          setCity(response.city);
+          setCountry(response.country);
+          setDatePosted(response.datePosted);
+          setClosingDate(response.closingDate);
+        });
+    }
+  }, []);
+  
+  const onSubmit = (event) => {
+    event.preventDefault();
+    let url;
+
+    const options = {
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        jobTitle: jobTitle,
+        clientId: clientId,
+        companyName: companyName,
+        jobDescription: jobDescription,
+        city: city,
+        country: country,
+        datePosted: datePosted,
+        closingDate: closingDate
+      })
+    };
+
+    if (PosId === null) {
+      options.method = 'POST';
+      url = `${process.env.REACT_APP_API}/api/positions/create`;
```

Comment:
If you check on the deployed app in [heroku](https://basd-2021-aq-ms-24-posi-owlhvx.herokuapp.com/positions) , the requests to the server are failing cause `/api` is duplicated. It is probably that `/api` already included in the 
`REACT_APP_API` environment variable. You should remove it from here and add it in your `.env` file

```suggestion
      url = `${process.env.REACT_APP_API}/positions/create`;
```